salt/states/boto_asg.py

```diff
@@ -652,6 +653,7 @@ def _recursive_compare(v1, v2):
 def absent(
         name,
         force=False,
+        remove_lc=False,
```

Comment:
@lomeroe, is it possible to move this argument to the end of the list, to keep api changes to a minimum?  Thanks.